Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -73,6 +75,11 @@ class Decoder;
 class FormDataReference;
 class SharedBufferReference;
 using DataReference = std::span<const uint8_t>;
+
+template<> struct AsyncReplyError<WebKit::WebEventType> {
+    static WebKit::WebEventType create() { return WebKit::WebEventType::NoType; }
+};
```

Comment:
But there is no `NoType` member of `WebKit::WebEventType`?

Reply:
Yeah, I think you either forgot to build this, or perhaps forgot to commit your changes to that file?

I'd suggest doing this in WebEventType.h instead, even if it requires including Connection.h, for extra safety.

Alternatively, perhaps it would be safe to simply change the default value, since MouseDown is not a good default. _Hopefully_ that won't break anything. If adding WebEventType::NoType you'll probably have to deal with a bunch of -Wswitch warnings anyway.

Reply:
Oh, I see. It was removed two days ago in https://github.com/WebKit/WebKit/commit/791e55b6738d6cf322148abc8c15fb39a61527cd.
That PR actually fixed the issue I tried to fix with this PR but in a more verbose way. I like my way more since it's more compact but it's probably too late. Should I close this one now?

Reply:
I've edited the PR to only update the `TestExpectations.json`.